packages/generator/src/options/options.ts

```diff
@@ -17,6 +21,10 @@ export interface GeneratorOptions extends CommonGeneratorOptions {
    * Number of node processes used for transpilation of JavaScript files.
    */
   transpilationProcesses?: number;
+  /**
+   * Whether to generate ECMAScript modules instead of CommonJS modules.
+   */
+  generateESM?: boolean;
```

Comment:
This should be moved to the common generator options since its duplicated in both generators